docs/articles/new-expensify/expensify-card/Add-Expensify-Card-to-Apple-or-Google-Pay.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Add Expensify Card to Apple or Google Pay
+description: Pay with your Expensify Card from your Apple or Google Pay wallet
+---
+<div id="new-expensify" markdown="1">
+
+You can use your Expensify Card for contactless in-person payments by adding it to your digital wallet for Apple Pay (for iOS) or Google Pay (for Android).
+
+{% include selector.html values="desktop, mobile" %}
```

Comment:
@ren-jones please remove the desktop option as we have no content for it.


```suggestion
{% include selector.html values="mobile" %}
```

![image](https://github.com/Expensify/App/assets/29673073/434f1b2e-8992-49bc-b397-68b472da0598)